CHANGELOG.md

```diff
@@ -5,6 +5,23 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## Release 5.0.1 (2020-10-20)
+
+[Changes since v5.0.0](https://github.com/realm/realm-studio/compare/v5.0.0...v5.0.1)
+
+### Enhancements
+
+- None
+
+### Fixed
+
+- Fixed JSON export (producing empty objects). ([#1344](https://github.com/realm/realm-studio/issues/1344))
```

Comment:
I think it's since 5.0.0, I don't see any other previous version uses `realm@6.1.0+`.

```suggestion
- Fixed JSON export (producing empty objects). ([#1344](https://github.com/realm/realm-studio/issues/1344), since v5.0.0)
```